src/main/java/svc/data/citations/datasources/importedITI/transformers/ImportedItiViolationTransformer.java

```diff
@@ -0,0 +1,39 @@
+package svc.data.citations.datasources.importedITI.transformers;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+import org.springframework.stereotype.Component;
+
+import svc.data.citations.datasources.importedITI.models.ImportedItiCitation;
+import svc.data.citations.datasources.importedITI.models.ImportedItiViolation;
+import svc.models.Violation;
+
+@Component
+public class ImportedItiViolationTransformer {
+
+	public List<Violation> fromImportedCitation(ImportedItiCitation importedItiCitation) {
+		if (importedItiCitation.violations != null) {
+			return importedItiCitation.violations.stream()
+					.map(importedItiViolation -> fromImportedItiViolation(importedItiCitation, importedItiViolation))
+					.collect(Collectors.toList());
+		}
+		return null;
```

Comment:
Same here `null` vs empty list

Reply:
I looked through the code here, and my memory is a little rusty here.  First all of the transformers follow this pattern of null vs empty list.  But I think this was intentional some time ago.  I think it has something to do with nothing being found triggers the correct error messages on the client code.  Null is indicative of nothing being found from that specific data source as well.